temporalcli/commands.go

```diff
@@ -89,6 +89,8 @@ func NewCommandContext(ctx context.Context, options CommandOptions) (*CommandCon
 	return cctx, stop, nil
 }
 
+const temporalEnv = "TEMPORAL_ENV"
```

Comment:
I don't think there's much value in making this a constant that is used in one place, but meh